vulture/core.py

```diff
@@ -163,6 +168,7 @@ def get_set(typ):
         self.used_names = get_set('name')
 
         self.ignore_names = ignore_names or []
+        self.ignore_decorators = ignore_decorators or []
```

Comment:
Strip leading @'s in next line and add tests for "@app.route" and "app.route".

Reply:
Hmm, so are we supporting both  `@app.route` and `app.route`??

Reply:
Ahh, I got it all wrong, stupid me.